packages/types/src/storages.ts

```diff
@@ -204,6 +204,7 @@ export interface RequestQueueInfo {
     actRunId?: string;
     hadMultipleClients?: boolean;
     stats?: RequestQueueStats;
+    forefrontRequestIds?: string[];
```

Comment:
What's the purpose of this one? It will only be there for local storage, right?

Reply:
Yes, this PR is only fixing the `MemoryStorage` implementation. `forefrontRequestIds` tracks the ordering of the forefront requests (without the need to reimplement the way everything else is handled). 

See `requestKeyIterator` (used in `listHead` to get the first N requests) for usage: https://github.com/apify/crawlee/blob/169d48bbfc1763cbbe990f3f64757d8697810d7b/packages/memory-storage/src/resource-clients/request-queue.ts#L156-L164

Reply:
Yes, I mean... it's internal state, why do we need to expose it via `getInfo()`?

Reply:
The original idea was to persist it with the other metadata, but you're right; we don't likely want to expose it.

Addressed in https://github.com/apify/crawlee/pull/2681/commits/adce71dea93debb1aa1247b15a1af86ce1ca3d67 .